client/state/initial-state.js

```diff
@@ -303,6 +303,11 @@ export function getStateFromCache( reducer, subkey ) {
 		reduxStateKey = getPersistenceKey( subkey, true );
 		persistedState = stateCache[ reduxStateKey ] ?? null;
 
+		// If we are logged in, we no longer need the 'user' step in signup progress tree.
+		if ( persistedState && persistedState.progress && persistedState.progress.user ) {
```

Comment:
You could use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining):

`persistedState?.progress?.user`

Reply:
Thanks!

For some reason, I still don't prefer this style.. but not even sure if that's a stylistic preference or the way the JS community goes forward. I guess I need to use it more to start liking it more 😅

Reply:
Yeah, I'd strive for more consistency within the codebase over personal preferences, which then means using this. :-)

Alternative of course is `lodash.has()` or `.get` but those aren't great since native alternative is available.